conf/v0.6/general/test/gpt.toml

```diff
@@ -0,0 +1,5 @@
+name = "gpt"
+description = "gpt"
+test_template_name = "NeMoLauncher"
+
+[cmd_args]
```

Comment:
I believe you can remove this section completely because it does not have any cmd args.